includes/Core/Util/Tracking.php

```diff
@@ -80,6 +80,9 @@ public function is_active() {
 	 * @return array Filtered $data.
 	 */
 	private function inline_js_base_data( $data ) {
+		global $hook_suffix;
+		
+		$data['trackingAllowed'] = false !== strpos( $hook_suffix, 'googlesitekit' );
```

Comment:
We should be specific to only match screens starting with `googlesitekit-`
```suggestion
		$data['trackingAllowed'] = 0 === strpos( $hook_suffix, 'googlesitekit-' );
```